openandroidinstaller/views/select_view.py

```diff
@@ -250,6 +362,7 @@ def pick_image_result(self, e: FilePickerResultEvent):
             self.selected_image.value.split(":")[0] + f": {path}"
         )
         if e.files:
+            # Why two variables needed?
```

Comment:
It's not my code and don't look deeply, but are these two variables needed ? It seems that only `self.state.image_path` is used...